docs/cli/installation.mdx

```diff
@@ -19,8 +27,8 @@ pagination_prev: null
 
 Install the latest version of the KNDP command-line via shell script by downloading the install script from KNDP GitHub Releases.
 Downloading packages requires both the version and CPU architecture (linux-amd64).
+VERSION=<Version/>
 ```bash
-VERSION="0.0.7" 
 ARCH="linux-amd64"
```

Comment:
@ca7alindev version shall be in bash section, it was specially requested for copy/paste install command with last version.

Reply:
@evghen1 but VERSION is read as string in bash section

Reply:
@ca7alindev Is outside of bash. What you mean?

Reply:
@evghen1 i meant all data from bash section is interpreted as string, so putting something like a variable it will be printed as is

Reply:
@ca7alindev Yes, then move to separate component outside of this MDX and simple render full bash block as component, like this:
https://github.com/facebook/docusaurus/blob/main/website/src/components/UpgradeGuide/index.tsx#L22
https://github.com/facebook/docusaurus/blob/main/website/docs/installation.mdx?plain=1#L156